modules/security-context-constraints-about.adoc

```diff
@@ -243,12 +250,22 @@ pre-allocated values. Uses the minimum value of the first range as the default.
 Validates against the first ID in the first range.
 * `RunAsAny` - No default provided. Allows any `fsGroup` ID to be specified.
 
-ifndef::openshift-dedicated[]
 [id="authorization-controlling-volumes_{context}"]
+ifndef::openshift-dedicated[]
 == Controlling volumes
-
-The usage of specific volume types can be controlled by setting the `volumes`
-field of the SCC. The allowable values of this field correspond to the volume
+endif::openshift-dedicated[]
+ifdef::openshift-dedicated[]
+== Controlling volumes for CCS clusters
+endif::openshift-dedicated[]
+
+The usage of specific volume types 
+ifdef::openshift-dedicated[]
+for {product-title} with Customer Cloud Subscription (CCS) clusters 
+endif::openshift-dedicated[]
+can be controlled by setting the `volumes`
+field of the SCC.
+
+The allowable values of this field correspond to the volume
 sources that are defined when creating a volume:
```

Comment:
Does it make sense to include links/options for volume values that are not supported on ROSA in the ROSA docs?  Below this we list things like Azure Disk/Files, iSCSI, GCE Persistent Disk, etc which cannot be used with ROSA.

Reply:
@0kashi thank you for your review and for highlighting this. I have subsequently asked the SRE team for clarification on which of the listed allowable values for volume sources are relevant for OSD and ROSA deployments specifically.

Reply:
@0kashi hi! As discussed during our previous call, can you please clarify which of the volume sources are supported with OSD and ROSA? Thanks!

Reply:
I'd say that for ROSA it would be configMap, downwardAPI, emptyDir, persistentVolumeClaim, secret, projected, awsElasticBlockStore and nfs. We should double check.